server/athenian/api/internal/miners/types.py

```diff
@@ -255,6 +272,27 @@ class DeploymentConclusion(IntEnum):
 s = None
 
 
+@dataclass(frozen=True, slots=True)
+class PullRequestJIRADetails:
+    """Extra JIRA information loaded for pull requests."""
+
+    ids: npt.NDArray[object]
+    projects: npt.NDArray[np.bytes_]
+    priorities: npt.NDArray[np.bytes_]
+    types: npt.NDArray[np.bytes_]
+
+    @classmethod
+    def empty(cls) -> "PullRequestJIRADetails":
+        """Return an empty instance."""
+        # return cls(*([None] * 4))
+        return cls(
+            ids=np.array([], dtype=object),
+            projects=np.array([], dtype="S"),
+            priorities=np.array([], dtype="S"),
+            types=np.array([], dtype="S"),
```

Comment:
It would be nice to set the exact sizes. IDK how smart is numpy in deducing the type inside `concatenate()`, but better to reduce the possibility of memory copies. Take them from `Issue.column.info["dtype"]`